AmazonChimeSDK/AmazonChimeSDK/device/DefaultDeviceController.swift

```diff
@@ -109,4 +108,20 @@ import AVFoundation
         let activeCamera = MediaDevice.fromVideoDevice(device: videoClientController.getCurrentDevice())
         return activeCamera.type == .other ? nil : activeCamera
     }
+
+    public func getActiveAudioDevice() -> MediaDevice? {
```

Comment:
PR changes lgtm. Can we add unit tests for newly added interfaces?

Reply:
done